docs/docs/cmd/graph/changelog/changelog-list.md

```diff
@@ -51,3 +51,52 @@ Get all changes within Microsoft Graph that happend between _2021-01-01_ and _20
 ```sh
 m365 graph changelog list --startDate '2021-01-01' --endDate '2021-05-01'
 ```
+
+## Response
+
+=== "JSON"
+
+    ```json
+    [
+      {
+        "guid": "8a74709d-d633-4cc1-81e2-89a67f4ae290v1.0",
+        "category": "v1.0",
+        "title": "Teamwork and communications",
+        "description": "Added the [getVirtualAppointmentJoinWebUrl](https://learn.microsoft.com/en-us/graph/api/virtualappointment-getvirtualappointmentjoinweburl?) function.\r\\\n",
+        "pubDate": "2023-05-23T00:00:00.000Z"
+      }
+    ]
+    ```
+
+=== "Text"
+
+    ```text
+    category  title                                          description
+    --------  ---------------------------------------------  --------------------------------------------------
+    v1.0      Teamwork and communications                    Added the getVirtualAppointmentJoinWebUrl (http...
+    ```
+
+=== "CSV"
+
+    ```csv
+    guid,category,title,description
+    8a74709d-d633-4cc1-81e2-89a67f4ae290v1.0,v1.0,Teamwork and communications,"Added the [getVirtualAppointmentJoinWebUrl](https://learn.microsoft.com/en-us/graph/api/virtualappointment-getvirtualappointmentjoinweburl?) function."
+    ```
+
+=== "Markdown"
+
+    ```md
+    # graph changelog list 
+
+    Date: 2023-05-22
+
+    ## Teamwork and communications
+
+    Property | Value
+    ---------|-------
+    guid | 8a74709d-d633-4cc1-81e2-89a67f4ae290v1.0
+    category | v1.0
+    title | Teamwork and communications
+    description | Added the [getVirtualAppointmentJoinWebUrl](https://learn.microsoft.com/en-us/graph/api/virtualappointment-getvirtualappointmentjoinweburl?) function.
+    <br>
```

Comment:
Is this <br> part of the response?

Reply:
Yes, <br> is part of the response. Should I remove it?

![image](https://github.com/pnp/cli-microsoft365/assets/18114579/7275899b-4ae8-4db7-aa43-0abd9ad2b416)

Reply:
Hmm weird, for me there's still content behind the <br> tag. Do you have an example that doesn't end on a `br` tag? If so, maybe we should use that example instead.

![image](https://github.com/pnp/cli-microsoft365/assets/11723921/b38e41f7-9b62-4c14-9d98-397f4decb3dc)